packages/@react-spectrum/calendar/stories/Calendar.stories.tsx

```diff
@@ -78,6 +79,14 @@ storiesOf('Date and Time/Calendar', module)
   .add(
     'minValue, visibleMonths: 3, defaultValue',
     () => render({minValue: new CalendarDate(2019, 6, 1), defaultValue: new CalendarDate(2019, 6, 5), visibleMonths: 3})
+  )
+  .add(
+    'defaultFocusedValue',
```

Comment:
just me being picky
it's almost a bit weird for it to be called focused value, I know why it is called that, but I don't think I'd look for "defaultFocusedValue" if I wanted to set "calendar should display this day/month/year first"

Reply:
suggestions welcome!!!

Reply:
i know, i wrote like 5 and discarded all of them!!! because it's doing what it says, but the side effect is that it shows that date first, and that's what people probably want to accomplish is the side effect